src/UI/Art/SpriteArtist/SpriteCOSetupArtist.java

```diff
@@ -161,12 +161,14 @@ private static void drawCoInfo(Graphics g, COSetupInfo info, int xCenter, int yC
       g.fillRect(drawX - drawScale, drawY - drawScale, drawW + (2*drawScale), drawH + (2*drawScale));
 
       // draw the CO's color selection
-      BufferedImage colorFrame = SpriteUIUtils.makeTextFrame(c, c.darker(), UIUtils.getColorName(c), 2*drawScale, 2*drawScale);
+      String palette = UIUtils.getPaletteDisplayName(UIUtils.getPaletteName(c), info.getCurrentFaction().name);
+      BufferedImage colorFrame = SpriteUIUtils.makeTextFrame(c, c.darker(), palette, 2*drawScale, 2*drawScale);
```

Comment:
Instead of calling `c.darker` repeatedly, you should just call it once and save/reuse it.

Reply:
Actually, belay that. No need to change this file. Because reasons.